tests/org.eclipse.core.tests.harness/META-INF/MANIFEST.MF

```diff
@@ -9,10 +9,11 @@ Bundle-Vendor: Eclipse.org
 Export-Package: org.eclipse.core.tests.harness,
  org.eclipse.core.tests.session,
  org.eclipse.core.tests.session.samples
-Require-Bundle: org.eclipse.core.runtime.compatibility,
- org.junit,
+Require-Bundle: org.junit,
  org.eclipse.pde.junit.runtime,
- org.eclipse.test.performance;resolution:=optional
+ org.eclipse.test.performance;resolution:=optional,
+ org.eclipse.osgi;bundle-version="3.12.50",
+ org.eclipse.core.runtime;bundle-version="3.13.0"
```

Comment:
Do we need version specific?

Reply:
I think this has been added by default. Thanks for noticing, will remove version tag.